apps/web/src/components/marketing/landing/customer/featured/index.tsx

```
@@ -3,36 +3,40 @@ import DotPattern from "@/components/ui/dot-pattern";
+              {t("title").split("</accent>")[1]}
             </Balancer>
           </h1>
 
-          <h2 className="text-center text-muted-foreground">
-            Parce que le meilleur c’est nous, <strong>onRuntime Studio</strong>,
-            votre allié pour réaliser cet objectif grâce à nos équipes de
-            passionnés.
-          </h2>
+          <h2
+            className="text-center text-muted-foreground"
+            dangerouslySetInnerHTML={{ __html: t("description") }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)